templates/index.html

```diff
@@ -41,18 +41,13 @@ <h2>Welcome to<span>a new dimension</span></h2>
                         </div>
                     </div>
                     <div class="col-sm-1 col-xs-1">
-                        <img class="planet-pink" src="{{ base_url }}static/img/planet-pink.png" alt="">
+                        <img class="planet-pink" src="{{ base_url }}static/img/planet-pink.svg" alt="">
                     </div>
                 </div>
             </article>
         </section>
     </div>
-    <section class="section-container">
-        <div class="row">
-            <div class="col-sm-12 col-sm-offset-0 col-xs-8 col-xs-offset-2">
-                <div class="soon-title">More information soon!</div>
-            </div>
-        </div>
-    </section>
-
+    {% if 'timeline' in this.sections %}
```

Comment:
any special reason the rendering is optional ?

Reply:
![Screen Shot 2020-08-10 at 8 34 17 AM](https://user-images.githubusercontent.com/14989202/89788161-5748aa00-dae4-11ea-8e16-0c79a33ad062.png)

In the landing page/index, we are using checkboxes to display some specific sections in order to preserve the source code of those sections